Draft-Accepted/Install-PowerShell.md

```diff
@@ -0,0 +1,217 @@
+  numbers should begin with 7.2
+- Parameter: Scope - overrides the default and allows the user to specify the installation location
+  (scope) AllUsers or CurrentUser.
+- Parameter: Channel - options Stable/Preview/LTS - allows the user to select if they want the
+  current preview or Long Term Service (LTS) release
+- Parameter: Passthru - to provide informative output post installation
+
+## Syntax and Parameter Sets
+
+### ParameterSet Version (Default)
+
+```
+Install-PowerShell7 [[-Version] <string>] [-Scope {AllUsers | CurrentUsers}] [-Passthru] [-WhatIf] [-Confirm]  [<CommonParameters>]
+```
+
+### ParameterSet Channel
+
+```
+Install-PowerShell7 -Channel {Preview | LTS | Stable} [-Scope {AllUsers | CurrentUser}] [-Passthru] [-WhatIf] [-Confirm]
+[<CommonParameters>]
+```
+
+The **Version** parameter has the following validation attribute: [ValidatePattern("^7\.\d+")]
+
+The **Channel** parameter has the following validation attribute: [ValidateSet("Preview", "LTS", "Stable")]
+
+The **Scope** parameter has the following validation attribute: [ValidateSet("AllUsers", "CurrentUser")]
+
+### Data model
+
+The `Install-PowerShell7` cmdlet should support the **Passthru** parameter to provide useful
+information post installation. The data model for the output is below;
+
+```output
+PSTypename   = "PSInstallationInfo"
+[Version]Version      = $version
+[string]Path         = (Get-Command pwsh).source
+[string]Channel      = if ($Channel) { $channel} else {"Stable" }
+[string]Computername = $env:COMPUTERNAME
+```
+
+## Demo.txt
+
+- Install the latest stable version by default.
+
+  ```powershell
+  install-powershell7 -Passthru
+  ```
+
+  ```
+  Version Path                                   Channel Computername
+  ------- ----                                   ------- ------------
+  7.2.3   C:\Program Files\PowerShell\7\pwsh.exe Stable  LocalComp
+  ```
+
+- To install the latest stable release of PowerShell over PowerShell Remoting:
+
+  ```powershell
+  Invoke-Command -Computername RemoteComputer1 -ScriptBlock {Install-PowerShell7}
+  ```
+
+- Install the LTS release using the current user scope.
+
+  ```powershell
+  Install-PowerShell7 -Passthru -Channel LTS -Scope CurrentUser
+  ```
+
+  ```
+  Version Path                                   Channel Computername
+  ------- ----                                   ------- ------------
+  7.2.0   C:\Users\jeff\PowerShell\7\pwsh.exe    LTS     RemoteComp
+  ```
+
+- To install the latest stable release of PowerShell:
+
+  ```powershell
+  Install-PowerShell7
+  ```
+
+- To install a specific version of a PowerShell release:
+
+  ```powershell
+  Install-PowerShell7 -Version 7.2
+  ```
+
+- To install a specific version of a PowerShell release to a specific scope:
+
+  ```powershell
+  Install-PowerShell7 -Version 7.2 -Scope Allusers
+  ```
+
+- To install the latest preview from a specific version:
+
+  ```powershell
+  Install-PowerShell7 -Version 7.2 -Channel 'Preview'
+  ```
```

Comment:
On a side note, we should consider updating our "new version available" banner to denote if the current version is EOL and out of support. Seems like it would be very important on Windows were the OS is supported for more than PS 7 support cycle.

Reply:
Good point that I would like to further discuss on the banner topic outside of this cmdlet.